dev/test/util/helpers.ts

```diff
@@ -342,6 +342,19 @@ export const postConverter = {
   },
 };
 
+export const postConverterMerge = {
+  toFirestore(post: Partial<Post>): DocumentData {
```

Comment:
Could we add SetOptions here and additional asserts? Something like that:

```
if (merge) {
expect(post).to.be.an.instanceOf(Post);
} else {
expect(post).to.not.be.an.instanceOf(Post);
}
```

Reply:
Done, but flipped the expect statements.